### PR TITLE
Better kill in fast test

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: docker/test/fasttest/run.sh [hunk 1]
```diff
@@ -163,15 +163,15 @@ clickhouse-test -j 4 --no-long --testname --shard --zookeeper --skip ${TESTS_TO_
...
 
     for i in {1..10}
     do
-        if ! kill -0 `ps ax | grep clickhouse-server | grep -v 'grep' | awk '{print $1}'`; then
+        if ! killall -0 clickhouse-server; then
             echo "No clickhouse process"
             break
         else
-            echo "Process" `ps ax | grep clickhouse-server | grep -v 'grep' | awk '{print $1}'` "still alive"
+            echo "Clickhouse server process" `pgrep -f clickhouse-server` "still alive"
```
Comment (akuzm) [inline docker/test/fasttest/run.sh:174]: shellcheck is going to complain about the backticks being deprecated and suggest $(...) as a modern alternative.

**Comment by akuzm** [comment]
Maybe just `ln -s /var/log/clickhouse-server /test_output`? This way you'll always have the log, even if the script fails in some unpredictable way.

**Comment by akuzm** [comment]
> Maybe just `ln -s /var/log/clickhouse-server /test_output`? This way you'll always have the log, even if the script fails in some unpredictable way.

I confused the argument order again... There is a good mnemonic -- this command is similar `cp`, so the arguments are in the same order, e.g. `cp <src> <dest>`, but it doesn't seem to help :)

**Review by akuzm — APPROVED** [review]

**Comment by alesapin** [comment]
fast test ok

**Pull request merged**